pages/admin/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import Layout from '../../components/layout';
+
+class Index extends React.Component {
+	state = {};
+
+	render() {
+		return (
+			<Layout selectedMenuItem="home">
+				<p>Banana</p>
+			</Layout>
+		);
+	}
+}
```

Comment:
index formatting on this file is off, make it look like the other react component files